src/main/java/org/shaolinmasters/akkadianlexicon/controllers/HomeController.java

```diff
@@ -1,16 +1,33 @@
 package org.shaolinmasters.akkadianlexicon.controllers;
 
 import lombok.RequiredArgsConstructor;
+import org.shaolinmasters.akkadianlexicon.models.WebContents;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.shaolinmasters.akkadianlexicon.services.HomeService;
+
+import java.util.List;
+import java.util.Optional;
 
-@Controller
 @RequiredArgsConstructor
+@Controller
 public class HomeController {
 
+  private final HomeService homeService;
+
   @GetMapping("/")
-  String getWelcome() {
-    return "welcome";
+  public String viewContent(Model model) {
+    Optional <String> title = "home_text".describeConstable();
+
+    //TODO: itt az lehhe a jo, hogy amikor az adatbazisba feltoltjuk a szoveget,
+    // akkor ezzel a cimmel vinnenk fel: home_text
+    // at is lehet nevezni, csak akkor itt es a html fajlban is at kell majd
+    //TODO: tovabbi teendo -> html-ben at kell irni az osztalyt, mert jelenleg hardcodolva adjuk vissza a szoveget
+
+    List<WebContents> aContent = homeService.findByTitle(title);
+    model.addAttribute("content", aContent);
+    return "home";
```

Comment:
- Mapping tokeletes. 
- Metodus nevnek en szerencsesebbnek gondolnam a home()vagy getHome() vagy getHomePage() elnevezeseket, de ez reszlet kerdes, akar maradhat igy is.
- A kovetkezo sorban nem tudom pontosan mi tortenik. Itt eleg sztem csak siman a harcode-olt home_text string-gel meghivni a servicunket, mert ez sosem fog valtozni, uh sztem az Optional-t elhagyhatod.
 (ezt altalaban akkor hasznaljuk, ha egy olyam metodus eredmenyet akarjuk elmenteni egy valtozoba, ami nem biztos, hogy visszaad nekunk elemet. Pl az adatbazistol elkerjuk azt a kiralyt, akinek a neve Nabukodonazor, de nem lehetunk benne biztosak, hogy az adatbazis valoban tartalmaz ilyen nevvel kiralyt, ezert annak a metodusnak az eredmenyet, ami elkeri az adatbazistol a kiralyt egy Optional<King> -be rakjuk bele, pl:
 `Optional<King> king = kingRepository.findKingByName(kingName);` )

- irtam a service osztalyhoz megjegyzest, hogy a findByTitle() metodus 
-> vagy egy WebContent-et ad vissza (abban az esetben, ha talal az adatbazisban a megadott title-lel bejegyzest)...ezt a WebContent-et aztan belerakjuk a model-be, amit atadunk a view-nak
-> vagy egy exception-t dob
Ezt az exception-t itt el kellene kapni, uh az egesz service hivast egy try-catch blockba erdemes belerakni.

- Erdemes idokozben a beerkezo kereseket, exceptionoket logolni is (errol annyira meg talan nem sok info hangzott el)


_A fentiek egy picit bonyolitjak ezt a metodust, igy ideirom az en megoldasomat, de a tobbi osztalynal ratok hagyom a javitast.....aztan persze ha elakadtok vele, akkor szivesen segitek_

```java
 @GetMapping("/")
  public String getHomePage(Model model) {
    logger.info("Incoming request for '/'");
    WebContent aContent;
    try {
      aContent = webContentService.findByTitle("home_text");
    } catch (RuntimeException runtimeException) {
      logger.error(runtimeException.getMessage());
      return "home";
    }
    model.addAttribute("content", aContent);
    return "home";
  }
  ```
  
  Ahhoz, hogy ez mukodjon, kell egy 
  `private final Logger logger = LoggerFactory.getLogger(HomeController.class);`
  az osztalyba.